test/cheerio.js

```diff
@@ -44,6 +44,13 @@ describe('cheerio', function () {
     expect($h2[0].tagName).toBe('h2');
   });
 
+  it('should be able to create html without a root or context, html tag', function () {
+    var $tag = cheerio('<html>');
+    expect($tag).not.toHaveLength(0);
```

Comment:
This is a little moot since you check for length 1 in the next line :)

Reply:
haha, I simply copied test above. I looked it, but since it was there I kept it.